src/postmeta/index.js

```diff
@@ -0,0 +1,8 @@
+const { registerPlugin } = wp.plugins;
```

Comment:
also here, and you need to npm install the plugins

Reply:
done